internal/repository/psql/account.go

```diff
@@ -0,0 +1,132 @@
+package psql
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/Viquad/crud-app/internal/domain"
+)
+
+type AccountRepository struct {
+	db *sql.DB
+}
+
+func NewAccount(db *sql.DB) *AccountRepository {
+	return &AccountRepository{
+		db: db,
+	}
+}
+
+func (b *AccountRepository) Create(ctx context.Context, account domain.Account) (*domain.Account, error) {
+	var id int64
+	var update time.Time
+	query := "INSERT INTO accounts (first_name, last_name, balance, currency) VALUES ($1, $2, $3, $4) RETURNING id, last_update"
+	err := b.db.QueryRowContext(ctx, query, account.FirstName, account.LastName, account.Balance, account.Currency).
```

Comment:
please change QueryRowContext to ExecContext in all DB methods which create, update and delete

Reply:
@tarkovskynik I used `QueryRowContext` here cause I added above `RETURNING id, last_update` in sql-query. This allows me to send the user a newly created account with all available fields.